packages/aws/package.json

```diff
@@ -13,12 +13,6 @@
     "directory": "packages/aws"
   },
   "scripts": {
-    "prettier": "prettier --check src/",
-    "prettier:fix": "prettier --write src/",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "pnpm lint --fix",
-    "verify": "run-p prettier lint",
-    "verify:fix": "pnpm prettier:fix && pnpm lint:fix",
```

Comment:
Was able to remove a bunch of scripts in the packages since everything is run from the root now